prow/cmd/deck/main.go

```diff
@@ -551,6 +556,13 @@ func prodOnlyMain(cfg config.Getter, o options, mux *http.ServeMux) *http.ServeM
 			if err != nil {
 				logrus.WithError(err).Fatal("Error getting GitHub client.")
 			}
+			if o.pluginConfig != "" {
+				pluginAgent := plugins.ConfigAgent{}
+				if err := pluginAgent.Load(o.pluginConfig); err != nil {
+					logrus.WithError(err).Fatal("Error loading Prow plugin config.")
+				}
+				pcfg = pluginAgent.Config()
```

Comment:
We need to use `pluginAgent.Config()` to get the config when we are serving the request so that we don't have to restart `deck` in order to pick up changes to the plugin config.